compiler/shared_ast/expr.mli

```diff
@@ -341,6 +341,8 @@ val remove_logging_calls :
     their argument. *)
 
 val format :
+  ?hide_function_body:bool
+    (** if [true], prints "<function>" for [EAbs] nodes *) ->
```

Comment:
Ah that will conflict with my PR that simplified the expr printing interface.
To be consistent we probably want to keep this in `Print.expr` (which has an added `unit` arg to make the optional arguments usable) and discard them in the simpler `Expr.format` (which fits more nicely in "%a" fmts), if that sound OK ?